OpenOversight/app/main/views.py

```diff
@@ -450,6 +451,8 @@ def upload(department_id):
     file_to_upload = request.files['file']
     if not allowed_file(file_to_upload.filename):
         return jsonify(error="File type not allowed!"), 415
+    if requires_conversion(file_to_upload.filename):
+        convert_image()
```

Comment:
It looks like the image is getting converted and saved on the filesystem but then we don't use the result and instead continue to use the image data stored in `file_to_upload.filename` (also in `convert_image` we actually remove the contents of `file_to_upload.filename`). I think we'll want to return the filename in `convert_image` and then use that image for the remainder of the code in `upload`. 

However -